subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c

```diff
@@ -174,7 +178,7 @@ static int prepare_cb(struct lll_prepare_param *p)
 
 	/* setup tIFS switching */
 	radio_tmr_tifs_set(EVENT_IFS_US);
-	radio_switch_complete_and_tx(lll->phy, 0, lll->phy, 1);
+	radio_switch_complete_and_tx(lll->phy, 0, lll->phy, 0);
```

Comment:
Since we haven't been using proper names for the flags in the code, can you elaborate on why we are changing the tx flag from 1 to 0 in a PR that deals with scanning?